hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapRemoveAllMessageTask.java

```diff
@@ -52,36 +49,19 @@ protected void processMessage() {
             super.processMessage();
             return;
         }
-
-        int partitionId = clientMessage.getPartitionId();
-
+        PartitionPredicate partitionPredicate = (PartitionPredicate) predicate;
         OperationFactory operationFactory = createOperationFactory();
         OperationServiceImpl operationService = nodeEngine.getOperationService();
 
-        // We are running on a partition thread now and we are not allowed
-        // to call invokeOnPartitions(Async) on operation service because of
-        // that.
```

Comment:
This is the only part I'm hesitant about, this comment indicates what I've done may not be desired.

Is there a better way to invoke on  partitions from here?

Reply:
I think it is a left-over comment, this part is not running on partition thread.
@mdumandag can you please confirm if this is the case?

Reply:
Yeah, this message task (and hence the `processMessage` method) is executed in the `hz:client` executor.